content/en/logs/faq/how-to-tail-logs-from-host-using-a-container-agent.md

```diff
@@ -0,0 +1,148 @@
+---
+
+<div class="alert alert-info">Datadog recommends using <b>STDOUT/STDERR</b> to collect container logs. This is the official documentation on collecting logs from <a href="/docker/log/?tab=containerinstallation#overview" target="_blank">Docker</a> and <a href="/agent/kubernetes/log/?tab=daemonset#pagetitle" target="_blank">Kubernetes</a>. To learn more about Docker logging, read the <a href="https://www.datadoghq.com/blog/docker-logging/" target="_blank">Docker Logging Best Practices</a> blog post.</div>
+
+## Overview
+
+Pods/containers have no access to host files by default, which also applies to the Agent. If you try to configure your container Agent to collect host files, you will see a similar error message to the one below:
+
+```
+  syslog
+  ------
+    Type: file
+    Path: /var/log/messages
+    Status: Error: file /var/log/messages does not exist
+
+```
+
+To give the Agent container access to host files, mount the file or its directory to the Agent container. Here are some examples for Kubernetes and Docker:
+
+### Kubernetes:
+
+To mount the log files in your host to the Agent container, set the host log directory in the volumes section of your Agent manifest and the container log directory in `volumeMounts` section:
+
+```
+        volumeMounts:
+          - name: customlogs
+            ## The desired log directory inside the agent container:
+            mountPath: /container/var/test-dir/logs/
+            
+      volumes:
+        - name: customlogs
+          hostPath:
+            ## The directory in your host containing the log files.
+            path: /var/test-dir/logs/
+```
+
+Next, configure the Agent to tail the files for log collection. To do this, you can mount a custom logs config into `/conf.d/`. The file name can be anything, as long as it has a `.yaml` extension.
+
+It is preferable to use a ConfigMap to store configurations rather than mounting a host file directly. Here's a sample ConfigMap manifest that has a `logs.yaml` file:
+
+```
+kind: ConfigMap
+apiVersion: v1
+metadata:
+     name: ddagent-logs-configmap
+     namespace: default
+data:
+     logs.yaml: |-
+           logs:
+             - type: file
+               service: syslog
+               source: os
+               ## Use the container log directory you set in the agent manifest
+               path: /container/var/test-dir/logs/*.log
+```
+
+Create the ConfigMap object using the command:
+
+```bash
+kubectl create -f <configmap manifest>
+```
+
+Then, mount it under `/conf.d/`:
+
+```
+        volumeMounts:
+          - name: cm-logs
+            mountPath: /conf.d/
+
+      volumes:
+        - name: cm-logs
+          configMap:
+            name: ddagent-logs-configmap
+```
+
+### Docker:
+
+To mount the host log file, you can add a volume parameter in your Agent's `docker run` command:
+
+```
+-v /<host log directory>/:<container log directory>/
+```
+
+Then, you will need to create a custom logs config locally:
```

Comment:
Same here, let's avoid `you will need`. The idea is to take it like a recipe. They don't tell you that you can add sugar but to add 100mg of sugar :)

```suggestion
Then, create a custom logs config locally:
```